### ISSUE TITLE
Test for WCAG failure F46: using th elements, caption elements, or non-empty summary attributes in layout tables

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#145]
http://www.w3.org/TR/WCAG20-TECHS/failures.html#F46

The objective of this technique is to describe a failure that occurs when a table used only for layout includes either th elements, a summary attribute, or a caption element. This is a failure because it uses structural (or semantic) markup only for presentation. The intent of the HTML and XHTML table elements is to present data.

Although not commonly used in a layout table, the following structural markup would also be failures of Success Criterion 1.3.1 if used in a layout table:

```
headers attributes

scope attributes 
```
## Tests

Procedure
1. Examine the source code of the HTML or XHTML document for the table element
2. If the table is used only to visually lay out elements within the content
   - [ ] Check that the table does not contain any th elements.
   - [ ] Check that the table element does not contain a non-empty summary attribute.
   - [ ] Check that the table element does not contain a caption element.
